pkg/queue/metrics.go

```diff
@@ -12,28 +12,28 @@ type Metrics struct {
 	querierWaitTime   *prometheus.HistogramVec // Per querier wait time
 }
 
-func NewMetrics(subsystem string, registerer prometheus.Registerer) *Metrics {
+func NewMetrics(subsystem string, registerer prometheus.Registerer, metricsNamespace string) *Metrics {
```

Comment:
might make more sense to keep the strings together, have the registerer as the first or last param?

Reply:
Good idea. Done